app/components/chart-bar.js

```diff
@@ -22,36 +22,68 @@ export default Component.extend({
     const data = get(this, 'data');
     const dataOrArray = data?data:[];
     const svg = select(this.element);
-    const margin = {top: 20, right: 20, bottom: 30, left: 40};
-    const chartWidth = get(this, 'width') - margin.left - margin.right;
-    const chartHeight = get(this, 'height')- margin.top - margin.bottom;
+    const margin = {top: 10, right: 20, bottom: 30, left: 25};
+    const width = get(this, 'width');
+    const height = get(this, 'height');
+    const chartWidth = width - margin.left - margin.right;
     const color = scaleOrdinal(schemeCategory10);
 
-    const x = scaleBand().range([0, chartWidth]).padding(0.1);
-    const y = scaleLinear().range([chartHeight, 0]);
+    const x = scaleBand().range([0, chartWidth]).padding(0.4);
 
-    x.domain(dataOrArray.map(d => d.label));
-    y.domain([0, max(dataOrArray, d => d.total)]);
+    svg.attr('style', 'width:' + width +'px;height:' + height +'px;');
 
-    svg.append('g').attr('transform', `translate(${margin.left},${margin.top})`);
+    if (dataOrArray.length === 0) {
```

Comment:
Do we want to do this?  In my limited testing it would prevent the initial animation.  If we still get the animation even with this then we can restructure some stuff.